nettests/psiphon.go

```diff
@@ -27,22 +27,22 @@ type PsiphonTestKeys struct {
 // GetTestKeys generates a summary for a test run
 func (h Psiphon) GetTestKeys(tk map[string]interface{}) (interface{}, error) {
 	var (
-		err error
-		ok  bool
+		ok       bool
+		testKeys PsiphonTestKeys
 	)
-	testKeys := PsiphonTestKeys{IsAnomaly: false, Failure: ""}
 	if tk["failure"] != nil {
 		testKeys.IsAnomaly = true
-		testKeys.Failure, ok = tk["failure"].(string)
+		failure, ok := tk["failure"].(*string)
```

Comment:
I found out that this pattern needed to be changed when testing stunreachability

Reply:
Is the same change perhaps needed also for BootstrapTime?

Reply:
No: `BootstrapTime` is not nullable. The issue here is that casting to `*string` and casting to `string` is not the same and we need to use the precise type. As for `BootstrapTime` it's `float64` so we're good.